docs/cli-usage.md

```diff
@@ -46,7 +46,7 @@ aragon init [app-name] [boilerplate]
 - `boilerplate`: the Github repo name or alias for a boilerplate to set up your app. The currently available boilerplates are:
 
 	- `react`: this boilerplate contains a very basic Counter app and a webapp for interacting with it. It showcases the end-to-end interaction with an Aragon app, from the contracts to the webapp.
-	- `react-kit`: it is a variation of the `react` boilerplate that also comes with a DAO Kit which will allow for using your app to interact with other Aragon apps like the Voting app. You can read more about DAO Kits [here](kits-intro.md).
+	- `react-kit`: it is a variation of the `react` boilerplate that also comes with a DAO Template which will allow for using your app to interact with other Aragon apps like the Voting app. You can read more about DAO Templates [here](kits-intro.md).
```

Comment:
I think we shouldn't change the "Kit" convention just yet, as it'll need a push across many different repos (and a new CLI release) to align them to "template". We also need to decide on their exact naming ("DAO Template", "Aragon Template", "Organization Template", etc.).

cc @luisivan

Reply:
reverted 8b05e3c8ebf9d2b732e7b9476bc42f5eb5d04f39